devops/scripts/install_build_tools.sh

```diff
@@ -9,5 +9,10 @@ apt update && apt install -yqq \
       python3 \
       python3-distutils \
       python-is-python3 \
+      python3-pip \
+      zstd \
       ocl-icd-libopencl1 \
       vim
+
+pip3 install psutil
```

Comment:
Could you clarify what these are for, please?

Reply:
> Also add zstd to experiment with LLVM compression times and psutil, which is required to kill tests by timeout.

That’s to build containers once instead of twice